src/lib/components/Error/ErrorReporter.tsx

```diff
@@ -0,0 +1,23 @@
+import { useAtomValue } from 'jotai/utils'
+import { providerAtom } from 'lib/state/web3'
+import { useEffect } from 'react'
+import { EMPTY } from 'widgets-web3-react/empty'
+
+class IntegrationError extends Error {
+  constructor(message: string) {
+    super(message)
+    this.name = 'INTEGRATION ERROR'
+  }
+}
+
+const missingProviderError = new IntegrationError('Missing provider')
+
+export default function ErrorReporter() {
```

Comment:
What's the purpose of this component?

Reply:
rendering it alongside children at the lowest level of the `Widget` wrapper to trigger the error boundary. Will extend w/ more error tests as necessary.